src/narrowdown.h

```diff
@@ -207,7 +207,7 @@ class NarrowDown
 
     bool operator()(const Entry& entry, const std::string& key) const
     {
-      return key.compare(getKeyContent(entry)) > 0;
+      return key.compare(getKeyContent(entry)) >= 0;
```

Comment:
:+1: How did you catch this bug? Can we add a unit test for it?

Reply:
Just by reading the code. I haven't found any real bug caused by this. (but it is probably important for case where two consecutive keys are equal on very small zim files).